src/GUI/GUI.purs

```diff
@@ -61,7 +61,7 @@ siteBody =
                 [ HP.classes [ HH.ClassName "columns" ] ]
                 [ HH.div
                     [ HP.classes [ HH.ClassName "column", HH.ClassName "is-three-quarters" ] ]
-                    [ HH.slot_ _proof unit GP.proof unit ]
+                    [ HH.slot _proof unit GP.proof 0 ActivateModal ]
```

Comment:
This should still be keyed by `unit` instead of `0` since we always have a singular `proof` component. I do not think we are consistent about this.

Reply:
Yeah, I just looked at how it was done in other places. But I'll fix it here.